app/controllers/request_controller.rb

```diff
@@ -678,13 +678,18 @@ def make_request_summary_file(info_request)
       tmp_input.write(html_output)
       tmp_input.close
       tmp_output = Tempfile.new('foihtml2pdf-output')
-      output = AlaveteliExternalCommand.run(convert_command, tmp_input.path, tmp_output.path)
```

Comment:
Think we could make this work by changing the `convert_command` to preserve the interface here.

```yaml
# config/general.yml
HTML_TO_PDF_COMMAND: "/usr/local/bin/wkhtmltopdf -q"
```

If this works then we'll want to update `config/general.yml-example` so that it gets set like this for everyone, and also note in the changelog that this is needed.

Reply:
This won't work without additional changes as this would eval to false:

```ruby
convert_command = AlaveteliConfiguration.html_to_pdf_command
File.exist?(convert_command)
```

Reply:
How about something like:

```yaml
# Default
HTML_TO_PDF_COMMAND: "/usr/local/bin/wkhtmltopdf"

# Optional
HTML_TO_PDF_COMMAND: ["/usr/local/bin/wkhtmltopdf", "-q"]
```

```ruby
convert_command = Array(AlaveteliConfiguration.html_to_pdf_command)
cmd = convert_command.shift
File.exist?(cmd)
output = AlaveteliExternalCommand.run(cmd, convert_command, tmp_input.path, tmp_output.path)
```